variables.tf

```diff
@@ -10,6 +10,12 @@ variable "project_name" {
   default     = "terraform-mongodbatlas-atlas-basic"
 }
 
+variable "use_existing_project" {
+  description = "Flag that determines whether the module uses an existing project or creates a new one."
+  type        = bool
+  default     = false
+}
+
 # mongodbatlas_project_ip_access_list
```

Comment:
should we adjust docs on database_users to clarify that default value will not be present when using existing project?

Reply:
Added it on the global readme. However I could also add it in the description of the variable if its necessary / clearer

Reply:
Great, yes I think that would be good so we cover the case of users that go directly to docs of the input variables.